go-controller/pkg/ovn/base_network_controller_multicast.go

```diff
@@ -186,37 +190,37 @@ func deleteMulticastAllowPolicy(nbClient libovsdbclient.Client, ns string) error
 //   - one ACL dropping ingress multicast traffic to all pods.
 //
 // Caller must hold the namespace's namespaceInfo object lock.
-func (oc *DefaultNetworkController) createDefaultDenyMulticastPolicy() error {
+func (bnc *BaseNetworkController) createDefaultDenyMulticastPolicy() error {
 	// By default deny any egress multicast traffic from any pod. This drops
 	// IP multicast membership reports therefore denying any multicast traffic
 	// to be forwarded to pods.
 	// By default deny any ingress multicast traffic to any pod.
 	match := getMulticastACLMatch()
 	acls := make([]*nbdb.ACL, 0, 2)
 	for _, aclDir := range []aclDirection{aclEgress, aclIngress} {
-		dbIDs := getDefaultMcastACLDbIDs(mcastDefaultDenyID, aclDir, oc.controllerName)
+		dbIDs := getDefaultMcastACLDbIDs(mcastDefaultDenyID, aclDir, bnc.controllerName)
 		aclPipeline := aclDirectionToACLPipeline(aclDir)
 		acl := BuildACL(dbIDs, types.DefaultMcastDenyPriority, match, nbdb.ACLActionDrop, nil, aclPipeline)
 		acls = append(acls, acl)
 	}
-	ops, err := libovsdbops.CreateOrUpdateACLsOps(oc.nbClient, nil, acls...)
+	ops, err := libovsdbops.CreateOrUpdateACLsOps(bnc.nbClient, nil, acls...)
 	if err != nil {
 		return err
 	}
 
-	ops, err = libovsdbops.AddACLsToPortGroupOps(oc.nbClient, ops, types.ClusterPortGroupName, acls...)
+	ops, err = libovsdbops.AddACLsToPortGroupOps(bnc.nbClient, ops, bnc.getPortGroupName(types.ClusterPortGroupName), acls...)
 	if err != nil {
 		return err
 	}
 
 	// Remove old multicastDefaultDeny port group now that all ports
 	// have been added to the clusterPortGroup by WatchPods()
-	ops, err = libovsdbops.DeletePortGroupsOps(oc.nbClient, ops, legacyMulticastDefaultDenyPortGroup)
+	ops, err = libovsdbops.DeletePortGroupsOps(bnc.nbClient, ops, bnc.getPortGroupName(legacyMulticastDefaultDenyPortGroup))
```

Comment:
we don't need to prefix stale port group right? since it was deleted before the networkPrefix is defined

Reply:
in that case, it is no-op. Otherwise, secondary network controllers would be trying to delete the legacy port groups of default network, which I don't want to happen. Another way is to do it only when `if !bnc.IsSecondary()` , but seems something we want to avoid as much as possible?

I personally is fine with either way.